File: src/main/java/org/battlescribedata/model/Catalogue.java

package org.battlescribedata.model;


public class Catalogue extends DataFile {
    
    private String gameSystemId;
    private int gameSystemRevision;

    public Catalogue() {
    }

    public String getGameSystemId() {
        return gameSystemId;
    }

    public void setGameSystemId(String value) {
        this.gameSystemId = value;
    }

    public int getGameSystemRevision() {
        return gameSystemRevision;
    }

    public void setGameSystemRevision(int value) {
        this.gameSystemRevision = value;
    }
}

File: src/main/dotnetCore/dotnetCore/Models/Catalogue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace dotnetCore.Models
{
    [Serializable]
    public class Catalogue : DataFile
    {
        public string GameSystemId { get; set; }
        public int GameSystemRevision { get; set; }
    }
}
